template/src/App.js

```diff
@@ -1,23 +1,66 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import { useSelector, useDispatch } from 'react-redux';
+import { selectors, actions } from './features/counter';
 
 function App() {
+  const count = useSelector(selectors.selectCount);
+  const dispatch = useDispatch();
+
   return (
     <div className="App">
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
+        <p className="App-counter">
```

Comment:
Let's move all the counter UI into a `<Counter />` component inside of `features/counter`.